{{cookiecutter.project_slug}}/.gitlab-ci.yml

```diff
@@ -0,0 +1,131 @@
+# templates
+# ------------
+
+.deploy-template:
+  image: alpine:latest
+  script:
+    - echo "Deploying to ${CI_ENVIRONMENT_NAME}"
+  variables:
+    GIT_STRATEGY: none  # prevents fetching repo (we only need the artifact here)
```

Comment:
I think we should skip the deployment template. There is no standard way to do deployments, so we should just leave this to the user.

Reply:
The reason for this is that I wanted to show how to use a job template. I agree with you that there is no standard way for deployments, hence I only put in the echo here. But I would not like to remove all the deployment jobs, as this ci definition should provide some kind of guideline and also an overview over some common functionalities of gitlab ci.

Reply:
yeah I see... what bothers me though is that this code doesn't do anything right now. So it will show up in the GitLab pipeline, it will be displayed as successful and therefore indicate to the casual observer, that this project has a deployment script (for a dev and prod environment) and that it was executed successfully.
I would argue that most of our projects won't have an automated deployment. Maybe let's save this for a different cookiecutter template that focuses on engineering projects?